test/extended/util/disruption/controlplane/known_backends.go

```diff
@@ -215,7 +217,8 @@ func createAPIServerBackendSampler(clusterConfig *rest.Config, disruptionBackend
 	if err != nil {
 		return nil, err
 	}
-	backendSampler.WithUserAgent("openshift-origin-external-backend-sampler")
+	backendSampler = backendSampler.WithUserAgent("openshift-origin-external-backend-sampler")
```

Comment:
could you adjust this user-agent like this: https://github.com/openshift/origin/pull/27755/files#diff-ec9a13d07468752514f167c195090f13d8b7f9cff945a0f3fc43a01e1ce469b6R219 ?